frontend/src/modules/entitieslist/components/Entity.css

```diff
@@ -102,3 +102,21 @@
 .entity.warnings .status:before {
     color: #ffa10f;
 }
+
+.entity.sibling {
+    background-color: #4d5967;
```

Comment:
Please make sure this background color is overridden on hover with the hover background color.